cdbot/cogs/fun.py

```diff
@@ -319,7 +330,11 @@ def check(reaction, user):
         Pulls all quotes from a quotes channel into a PostgreSQL database.
         Needs PGHOST, PGPORT, PGUSER, PGDATABASE and PGPASSWORD env vars.
         """
-        conn = await asyncpg.connect()
+        conn = await asyncpg.connect(host=base64.b64decode(environ.get("PGHOST")),
```

Comment:
This is not nice, could this be stored in the constants file so you can pass it straight into the function?